modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -114,4 +118,32 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async decodeLnurlPay(lnurl: string): Promise<LnurlPayResponse & { domain: string }> {
+    const url = decodeLnurl(lnurl);
+    const body = await this.bitgo.get(url).result();
```

Comment:
I don't think we should use the `bitgo.get` here because it'll add headers and authorization that's specific to interaction with the BitGo API, whereas this is a non-BitGo server. Can we `import request from superagent` and do `request.get` here?

Reply:
FYI, I tried this branch before you made the change to superagent and this request was failing with `ApiResponseError: invalid response HMAC, possible man-in-the-middle-attack` because it wasn't a request to a BitGo server. With the latest I can get past that error.